lib/pages/chat/chat_view.dart

```diff
@@ -170,7 +170,8 @@ class ChatView extends StatelessWidget with MessageContentMixin {
                             sendController: controller.sendController,
                             connectivityResultStream: controller
                                 .networkConnectionService
-                                .getStreamInstance(),
+                                .connectivity
+                                .onConnectivityChanged,
```

Comment:
what is the different? IMO, it works well now.

Reply:
no real difference, just more straightforward since the package gives us directly this stream